From ed592e301e3d1ff5a9ff6b90124f23c95df7ec2e Mon Sep 17 00:00:00 2001
From: Todd Kjos <tkjos@google.com>
Date: Thu, 7 Nov 2019 11:17:41 -0800
Subject: [PATCH 584/830] ANDROID: Kconfig.gki: Add SND_SOC_COMPRESS
X-NVConfidentiality: public

Select SND_SOC_COMPRESS via GKI_HIDDEN_SND_SOC_CONFIGS

Bug: 144073903
Change-Id: Ia389f26788d28a33b14fe116328a86968cb2d365
Signed-off-by: Todd Kjos <tkjos@google.com>
---
 init/Kconfig.gki | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/Kconfig.gki b/init/Kconfig.gki
index c5a080e..ad9c434 100644
--- a/init/Kconfig.gki
+++ b/init/Kconfig.gki
@@ -23,6 +23,7 @@ config GKI_HIDDEN_SND_SOC_CONFIGS
 	bool "Hidden SND_SOC configs needed for GKI"
 	select SND_SOC_GENERIC_DMAENGINE_PCM if (SND_SOC && SND)
 	select SND_PCM_IEC958
+	select SND_SOC_COMPRESS if (SND_SOC && SND)
 	help
 	  Dummy config option used to enable hidden SND_SOC configs.
 	  These are normally selected implicitely when a module
-- 
2.7.4

